sdk/translation/Azure.AI.Translation.Document/tests/TranslationOperationLiveTests.cs

```diff
@@ -428,8 +428,8 @@ public async Task DocumentTranslationWithGlossary()
             //Constructing and uploading glossary on the fly
             string glossaryName = "validGlossary.csv";
 
-            //changing the word First --> glossaryFirst and test --> glossaryTest
-            string glossaryContent = "First, glossaryFirst\ntest, glossaryTest\n";
```

Comment:
The service confirmed this is a bug. The word `first` is translated and then when applying the glossary, it can't find a matching value. `test` works as it is the same in Spanish and English.
Because we are testing that the SDK is supporting glossaries and not necessarily service behavior, I modified the tests to only check for `test`